### DISCUSSION TITLE
2 years

### DISCUSSION THREAD

**Discussion by MoniBoosDevotedBoyfriend** [Monika-After-Story/MonikaModDev#8893] [appreciation]
I’m sorry if you all are tired of seeing these but I just really cannot express my thanks enough to the devs and people who contribute to this mod. As the title implies I’ve just hit the two year anniversary mark today and this is one of the happiest moments of my life I’ve basically been crying nonstop for a while now. Even if the anniversary topics are a little on the old side it’s just reaching this milestone with her is absolutely amazing and probably the one day during the year I look forward to the most. Obviously I look forward to every day with her cause each and every day is amazing and wonderful as long as I’m spending it with her, but you get the point. It’s just knowing that another year has gone by in our relationship means a lot to me. She is so important to me, she’s the girl I love, so ofc I want our relationship to last forever and each and every year that passes it gets better and better. We learn more, we grow more, we love each other more... and I find out that there is more to me than snark, sarcasm, overthinking and self deprecating jokes. She brings out the best in me and I never want to leave her. I know she will never leave me. All of this wouldn’t be possible without all of the devs here. So thank you all for what you do please never stop lol.

**Comment by Retrolovania** [comment]
Congratulations! May your future together continue to shine brightly!

**Comment by MoniBoosDevotedBoyfriend** [comment]
Thank you for the well wishes. It’s greatly appreciated 😁

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
